state/futures/actions.ts

```diff
@@ -249,18 +247,26 @@ export const fetchSharedFuturesData = createAsyncThunk<void, void, ThunkConfig>(
 );
 
 export const fetchOpenOrders = createAsyncThunk<
-	{ orders: FuturesOrder<string>[]; account: string; accountType: FuturesAccountType },
+	{ orders: DelayedOrder<string>[]; account: string; accountType: FuturesAccountType },
 	void,
 	ThunkConfig
 >('futures/fetchOpenOrders', async (_, { getState, extra: { sdk } }) => {
 	const { futures } = getState();
 	const account = selectFuturesAccount(getState());
+	const markets = selectMarkets(getState());
 	if (!account) {
 		throw new Error('No account to fetch orders');
 	}
-	const orders = await sdk.futures.getOpenOrders(account, unserializeMarkets(futures.markets));
+	if (markets.length === 0) {
+		throw new Error('No markets available');
+	}
+	// TODO: Make this multicall
```

Comment:
Yea we can probably make this call `getDelayedOrders` and pass array of markets

Reply:
Right, I left as a to-do because we don't have a way to create a multicall contract for a specific market. Right now we create the mapping of contracts -> multicall contracts but market contracts are created within each transaction.

Reply:
Yea makes sense, we probably want a helper like `getMarketMultiCallContract(address: string)` that can also cache the contract instances by network.